TdvfPkg/UefiPayload/AcpiPlatformDxe/AcpiPlatform.c

```diff
@@ -74,17 +53,11 @@ InstallAcpiMadtTable (
                          );
   }
   ASSERT (CpuCount >= 1);
-  //
-  // Set Level-tiggered, Active High for these identity mapped IRQs. The bitset
-  // corresponds to the union of all possible interrupt assignments for the LNKA,
-  // LNKB, LNKC, LNKD PCI interrupt lines. See the DSDT.
-  //
-  PciLinkIsoCount = CountBits16 (PcdGet16 (Pcd8259LegacyModeEdgeLevel));
 
   NewBufferSize = 1                     * sizeof (*Madt) +
                   CpuCount              * sizeof (*LocalApic) +
                   1                     * sizeof (*IoApic) +
-                  (1 + PciLinkIsoCount) * sizeof (*Iso) +
+                  16 * sizeof (*Iso) +
```

Comment:
May I know why use the magic number 16 here? And please align the format

Reply:
The number of legacy IRQ for intel 8259 PIC.
will fix the indent.

Reply:
Hi, Isaku
Do you have any update of the PR?

Reply:
I've updated it.